ibis/expr/types/generic.py

```diff
@@ -547,29 +547,29 @@ def isin(self, values: Value | Sequence[Value]) -> ir.BooleanValue:
         Check against a derived expression
 
         >>> t.a.isin(t.b + 1)
-        ┏━━━━━━━━━━━━━━━━━━━━━━━━┓
-        ┃ InColumn(a, Add(b, 1)) ┃
-        ┡━━━━━━━━━━━━━━━━━━━━━━━━┩
-        │ boolean                │
-        ├────────────────────────┤
-        │ False                  │
-        │ False                  │
-        │ True                   │
-        └────────────────────────┘
+        ┏━━━━━━━━━━━━━━━┓
+        ┃ InSubquery(a) ┃
```

Comment:
This is probably not the nicest name we could generate, in order to change that we can override the `name` attribute. Just mentioning here, since we should do that for more operations in a follow-up once.